### DISCUSSION TITLE
Icon size in sidebar

### DISCUSSION THREAD

**Discussion by wora55** [freeplane/freeplane#782] [Help]
Is there a possibility to change the icon size in the side bar? 
Changing the font size for my high resolution monitor worked fine in 1.10.4, but the icons are hard to distinguish.
A possible workout would be a website showing all the icons in the same order (the problem is mostly with the navigation, once I picked an icon for the mindmap  I see it clear enough

**Comment by euu2021** [comment]
I don't know if it helps, but you can also try picking the icons from :

- Help->Command Search

Or

- Insert->Icons->Icon from table…

**Comment by wora55** [comment]
Does not really help because the icon size is still the same (4mm on my screen in a distance of about 80cm).
I only asked to make sure I did not miss a simple way to enlarge the icons.
Thanks anyway



euu schrieb am Sonntag, 30. Oktober 2022 15:28:20 (+01:00):



I don't know if it helps, but you can also try picking the icons from :

Help->Command Search

Or

Insert->Icons->Icon from table…

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.


-- 
Wolfgang Radke
Schubertstr. 6f
76344 Eggenstein

**Reply by dpolivaev** [reply]
The icons should have the correct size if your monitor size is properly set in the freeplane preferences.
Preferences…->Appearance->Look and feel->Monitor size (inches)

**Comment by wora55** [comment]
Thanks. That fixed my problem!

Dimitry Polivaev schrieb am Montag, 31. Oktober 2022 21:07:24 (+01:00):



The icons should have the correct size if your monitor size is properly set in the freeplane preferences.
Preferences…->Appearance->Look and feel->Monitor size (inches)

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.


-- 
Wolfgang Radke
Schubertstr. 6f
76344 Eggenstein
